Xamarin.Forms.Core/ListView.cs

```diff
@@ -391,27 +384,17 @@ protected override SizeRequest OnSizeRequest(double widthConstraint, double heig
 		protected override void SetupContent(Cell content, int index)
 		{
 			base.SetupContent(content, index);
-			if (content is ViewCell viewCell && viewCell.View != null && HasUnevenRows)
+			var viewCell = content as ViewCell;
+			if (viewCell != null && viewCell.View != null && HasUnevenRows)
 				viewCell.View.ComputedConstraint = LayoutConstraint.None;
-
-			if (content != null)
-				_logicalChildren.Add(content);
-
 			content.Parent = this;
-			VisualDiagnostics.OnChildAdded(this, content);
```

Comment:
can we keep this line ?

Reply:
Yea... I think I misunderstood our conversation yesterday :-) I thought you were indicating that if we were getting rid of LogicalChildren then these lines had no purpose